client/src/pages/SignUp.js

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from "react";
+import BidderSignUp from "./BidderSignUp";
+import ContractorSignUp from "./ContractorSignUp";
+
+function SignUp() {
+	// To keep track value of the radio buttons
+	const [userType, setUserType] = useState("");
+
+	// Decides signup form to be shown or not
+	const [showForm, setShowForm] = useState(false);
+	// Radio buttons to be shown or hidden
+	const [showRadioForm, setShowRadioForm] = useState(true);
+
+	const handleSubmitUserType = (e) => {
+		e.preventDefault();
+		setShowForm(true);
+	};
+
+	const handleChangeRdaio = (e) => {
+		setUserType(e.target.value);
+	};
+
+	const handleBackButtonClick = () => {
+		setShowForm(false);
+		setShowRadioForm(true); // To show radio buttons
+	};
+
+	return (
+		<>
+			<div>
+				<div>
+					{showForm && <button onClick={handleBackButtonClick}>Back</button>}
+				</div>
+				{!showForm && showRadioForm && (
+					<form onSubmit={handleSubmitUserType}>
+						<p>I am a :</p>
```

Comment:
Users don't sign up themselves, they are signed up by an admin

Reply:
that's correct but even admin still need to select a type for the user . and render corresponding form for the admin. this what I meant to be , otherwise I can render only one form containing all the fields .

Reply:
It's fine if you decide to render a different form, it's the `I am a:` statement that is incorrect. Something like `User type:` would be more accurate.